src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/managed/ManagedStrategy.kt

```diff
@@ -219,37 +209,27 @@ abstract class ManagedStrategy(
         runner = createRunner()
         ManagedStrategyStateHolder.setState(runner.classLoader, this, testClass)
         runner.initialize()
+        initializeInvocation()
```

Comment:
Shouldn't it be in the `runInvocation` logic?

Reply:
Do you mean that we can remove `initializeInvocation` from the public interface `Strategy`, and instead just make it protected method of `ManagedStrategy`, which is called at the beginning of `runInvocation` ?

Reply:
Sounds good.